lib/administration/user/include/irods/user_administration.hpp

```diff
@@ -74,7 +74,14 @@ namespace irods::experimental::administration
     /// \since 4.3.1
     struct user_password_property
     {
-        std::string value;
+        std::string value;              ///< The new password.
+        std::optional<std::string> key; ///< The password of the user changing the password.
+
+        explicit user_password_property(std::string _value, std::optional<std::string> _key = std::nullopt)
```

Comment:
Document

Reply:
Unnecessary.

The constructor exists to enforce proper usage which is transparent to all readers of the type definition.